machinelearning/horovod/horovod.def

```diff
@@ -0,0 +1,111 @@
+BootStrap: docker
+From: nvidia/cuda:9.0-devel-ubuntu16.04
+# -----------------------------------------------------------------------------------
+# This is a port of the Dockerfile maintained at https://github.com/uber/horovod
+
+
+%environment
+# -----------------------------------------------------------------------------------
+
+    export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+    export LC_ALL=C
+    export HOROVOD_GPU_ALLREDUCE=NCCL
+    export HOROVOD_GPU_ALLGATHER=MPI
+    export HOROVOD_GPU_BROADCAST=MPI
+    export HOROVOD_NCCL_HOME=/usr/local/cuda/nccl
+    export HOROVOD_NCCL_INCLUDE=/usr/local/cuda/nccl/include
+    export HOROVOD_NCCL_LIB=/usr/local/cuda/nccl/lib 
+    export PYTHON_VERSION=2.7
+    export TENSORFLOW_VERSION=1.11.0
+    export PYTORCH_VERSION=0.4.1
+    export CUDNN_VERSION=7.3.1.20-1+cuda9.0
+    export NCCL_VERSION=2.3.5-2+cuda9.0
+
+%post
+# -----------------------------------------------------------------------------------
+# this will install all necessary packages and prepare the contianer
```

Comment:
Comment here: #this will install all necessary packages and prepare the **container**